ansible/roles/mn-evo-services/tasks/tendermint.yml

```diff
@@ -9,29 +9,37 @@
     owner: '{{ tendermint_uid }}'
     group: '{{ tendermint_gid }}'
     mode: 0750
-  register: tendermint_home_dir
 
-- name: initialize Tendermint node
-  command: docker-compose run --no-deps --rm --entrypoint='/usr/bin/tenderdash' tendermint init
-  args:
-    chdir: '{{ mn_evo_services_path }}'
-  when: tendermint_home_dir.changed
-
-- name: get Tendermint node ID
-  command: docker-compose run --no-deps --rm --entrypoint='/usr/bin/tenderdash' tendermint show-node-id
-  args:
-    chdir: '{{ mn_evo_services_path }}'
-  register: tendermint_node_id
+- name: create tendermint config dir
+  file:
+    path: '{{ mn_evo_services_path }}/tendermint/config'
+    state: directory
+    owner: '{{ tendermint_uid }}'
+    group: '{{ tendermint_gid }}'
+    mode: 0750
 
-- name: get Tendermint validator key
-  command: cat {{ mn_evo_services_path }}/tendermint/config/priv_validator_key.json
-  register: tendermint_validator_key
+- name: create tendermint data dir
+  file:
+    path: '{{ mn_evo_services_path }}/tendermint/data'
+    state: directory
+    owner: '{{ tendermint_uid }}'
+    group: '{{ tendermint_gid }}'
+    mode: 0750
 
-- set_fact:
-    tendermint_node_id: '{{ tendermint_node_id.stdout }}'
+- name: create Tendermint node_key.json
+  template:
+    src: 'roles/mn-evo-services/templates/tendermint/node_key.json.j2'
+    dest: '{{ mn_evo_services_path }}/tendermint/config/node_key.json'
+    owner: '{{ tendermint_uid }}'
+    group: '{{ tendermint_gid }}'
 
-- set_fact:
-    tendermint_validator_key: '{{ tendermint_validator_key.stdout }}'
+- name: create Tendermint priv_validator_key.json
+  template:
+    src: 'roles/mn-evo-services/templates/tendermint/priv_validator_key.json.j2'
+    dest: '{{ mn_evo_services_path }}/tendermint/config/priv_validator_key.json'
+    owner: '{{ tendermint_uid }}'
+    group: '{{ tendermint_gid }}' 
+  when: skip_validator is not defined or skip_validator == false
```

Comment:
I guess you can put these files for all nodes like `tendermint init` did previously. It doesn't matter.